app/js/lib/actions.js

```diff
@@ -114,6 +114,15 @@ module.exports = {
 			type: 'FINISH_SEPA_CONFIRMATION_VALIDATION',
 			payload: validationResult
 		};
+	},
+
+	newCountrySpecificsUpdateAction: function ( countryCode ) {
```

Comment:
Is this new action really needed? Couldn't the reducer just listen to the `CHANGE_CONTENT` action of the country field?